### PR TITLE
UAntwerpen minor update Sept 2018

### PR DESCRIPTION
**Pull request by sbecuwe**
intro-HPC/ch_introduction.tex
- removed old information (Scientific Linux till end 2017)
- updated available software

intro-HPC/ch_preparing_the_environment.tex
- updated contents of tutorial directory

intro-HPC/ch_running_batch_jobs.tex
- removed old information (7 days queue)

sites/antwerpen/contact-information.tex
- added Thomas

sites/antwerpen/queue-table.tex
- updated information

### CODE REVIEW

**Comment by boegel** [comment]
@sbecuwe We're moving fast thanks to @redfast00's work...

Can you fix the conflict with current `master`? We'll try and get this merged first before Jasper causes more 'trouble'...

File: intro-HPC/ch_preparing_the_environment.tex [hunk 1]
```diff
@@ -262,6 +265,7 @@ \section{First Time connection to the \hpc}
 %\item  The \strong{\emph{VI}} editor.
 \item  Introduction to using \strong{\emph{perfexpert}}.
 \end{enumerate}
+\fi
```
Comment (boegel) [inline intro-HPC/ch_preparing_the_environment.tex]: @sbecuwe Let's change this for all sites?

**Comment by boegel** [comment]
@sbecuwe Tests failed, maybe you didn't pull in latest `master` before updating the version?

**Comment by sbecuwe** [comment]
AFAIK I had the latest master before updating. I also checked, and it is still consistent with master. check-version.sh tells me the checksums for both intro-HPC and intro-Linux are still valid...

**Comment by boegel** [comment]
@sbecuwe Hmm, that's also what I'm seeing when looking at your branch, I'll try and figure out what's wrong (Travis test has just been re-triggered, maybe that's enough).

**Comment by boegel** [comment]
@sbecuwe The checksum mismatch was caused by different default for `sort` on Linux vs macOS, fixed by using `sort -f -u` in `check-version.sh` (see 9a3515c)

This problem was probably introduced when @redfast00 added `intro-HPC/ch_FAQ.tex`, which is sorted first on macOS, yet case-insensitive on Linux...

Fixed in #264 together with your updates, we'll get that merged when Travis gives green light (and then so will this PR).

**Comment by boegel** [comment]
merged via #264

**Pull request merged**